Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_5.yml

```diff
@@ -0,0 +1,3169 @@
+    task:
+      id: c6980b2d-260a-4978-81c6-5ac79f5a6984
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -247.5,
+          "y": 4745
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 8a0081a4-4e44-4e1c-80bc-13d9bdc89105
+    type: title
+    task:
+      id: 8a0081a4-4e44-4e1c-80bc-13d9bdc89105
+      version: -1
+      name: Email Is Malicious
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "149"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 432.5,
+          "y": 3070
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: 4d2efdea-5293-424e-8ba3-49c2192a83a4
+    type: title
+    task:
+      id: 4d2efdea-5293-424e-8ba3-49c2192a83a4
+      version: -1
+      name: Undetermined
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "7"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -920,
+          "y": 3070
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: 3945c2b1-79f8-4572-80b6-bde4a220f534
+    type: condition
+    task:
+      id: 3945c2b1-79f8-4572-80b6-bde4a220f534
+      version: -1
+      name: Is the email malicious?
+      description: Is the email that the user reported malicious?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "No":
+      - "151"
+      "yes":
+      - "149"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -920,
+          "y": 3380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "34":
+    id: "34"
+    taskid: 8f9cdca2-c3cc-408b-8fc8-7620f3ac65be
+    type: regular
+    task:
+      id: 8f9cdca2-c3cc-408b-8fc8-7620f3ac65be
+      version: -1
+      name: Manually search & delete emails
+      description: Search for and delete emails with similar attributes of the malicious email using your existing email platfrom. for example - EWS, Office 365, Gmail etc.
```

Comment:
I actually was suggesting on adding some examples for using XSOAR commands/playbooks.
If that is too much info in one section, we can keep it.